media-libs/speexdsp/speexdsp-1.2~rc3.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A patent-free, Open Source/Free Software DSP library"
+DESCRIPTION="Speex is an open-source/free software  patent-free audio \
+compression format designed for speech. The Speex project aims to lower the \
+barrier of entry for voice applications by providing a free alternative to \
+expensive proprietary speech codecs. Moreover, Speex is well-adapted to \
+internet applications and provides useful features that are not present in \
+most other codecs.
+Finally, Speex is part of the GNU project and is available under the revised \
+BSD license."
+HOMEPAGE="https://www.speex.org/"
+COPYRIGHT="2002-2009 Xiph.org Foundation
+	2002-2008 Jean-Marc Valin
+	2005-2007 Analog Devices Inc.
+	2005-2008 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+	1993, 2002, 2006 David Rowe
+	2003 EpicGames
+	1992-1994 Jutta Degener, Carsten Bormann"
+LICENSE="Speex"
+REVISION="1"
+SOURCE_URI="https://downloads.xiph.org/releases/speex/speexdsp-${portVersion/\~/}.tar.gz"
+CHECKSUM_SHA256="4ae688600039f5d224bdf2e222d2fbde65608447e4c2f681585e4dca6df692f1"
+SOURCE_DIR="speexdsp-${portVersion/\~/}"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="1.5.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	speexdsp$secondaryArchSuffix = $portVersion
+	lib:libspeexdsp$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	speexdsp${secondaryArchSuffix}_devel = $portVersion
+	devel:libspeexdsp$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	speexdsp$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage speexdsp$secondaryArchSuffix \
+	"$libDir"/libspeexdsp.so.$libVersion
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -f "$libDir"/*.la
+
+	prepareInstalledDevelLib libspeexdsp
```

Comment:
```diff
 	rm -f "$libDir"/*.la
 
+	if [ -n "$secondaryArchSuffix" ]; then
+		rm -rf "$docDir"
+	fi
+
 	prepareInstalledDevelLib libspeexdsp
 	fixPkgconfig
```
This will drop the `documentation/packages/speexdsp/manual.pdf` on secondary arch builds.
(Remark, for our visitors reading this PR: it is OK to do so because the recipe builds on all primary architectures.)

BTW, shall we add a `TEST()` function?
```diff
 }
+
+TEST()
+{
+	make check
+}
```
(There does not seem to be any real test, but having the `TEST()` function here is slightly better, imo, than not having it, because this lets us know, at least, that there is a `check` target in the Makefile. What do you think?)